develop/dev-guide-delete-data.md

```diff
@@ -323,3 +324,39 @@ In each iteration, `DELETE` deletes up to 1000 rows from `2022-04-15 00:00:00` t
 </div>
 
 </SimpleTab>
+
+## Non-Transactional bulk-delete
+
+> **Note:**
+>
+> Since v6.1.0, TiDB supports the [Non-Transactional DML statements](/non-transactional-dml.md). This feature is not available for versions earlier than TiDB v6.1.0.
```

Comment:
```suggestion
> Since v6.1.0, TiDB supports the [non-transactional DML statements](/non-transactional-dml.md). This feature is not available for versions earlier than TiDB v6.1.0.
```